src/l10n/MailboxTranslator.js

```diff
@@ -40,7 +40,11 @@ const translateSpecial = folder => {
 	}
 	if (folder.specialUse.includes('inbox')) {
 		// TRANSLATORS: translated mail box name
-		return t('mail', 'Inbox')
+		if (folder.isUnified === true) {
+			return t('mail', folder.name)
```

Comment:
This is a bit problematic. Translation strings are extracted statically from the source code.

Hence, could you please inline the `All inboxes` string so it's `return t('mail', 'All inboxes')`? Then we might also be able to remove the `name` property from the folder. I think it's not used anywhere else.